src/components/ContentExplorer/ItemList.js

```diff
@@ -89,8 +89,11 @@ const ItemList = ({
         isSmall, // shows details if false
         isTouch
     );
+    const modifiedCell = modifiedCellRenderer(
+    	rootId,
```

Comment:
I think having them in 1 line should not give eslint errors.
If you have eslint and prettier configured, that will re-arrange the file.

Reply:
I have used the same formatting as other cell renderers. As for eslint, I installed it and configured it to use the Air BnB style as in package.json. The .eslintrc.json has this:
{
    "extends": "airbnb"
}
I get a dependency resolutions error when I attempt to run eslint for this file.
The Travis CI build error were with the ItemModified.js and modifiedCellRenderer.js files. I am fixing the errors on those files. 
Is there a build tool that I can run that will report these errors?